modules/sw360/sw360-client/src/main/java/org/eclipse/sw360/antenna/sw360/client/rest/resource/releases/SW360Release.java

```diff
@@ -291,6 +293,18 @@ public SW360Release setProprietary(boolean proprietary) {
         return this;
     }
 
+    @JsonIgnore
+    public String getHomepageurl() {
+        return homepageurl;
+    }
+
+    @JsonIgnore
+    public SW360Release setHomepageurl(String homepageurl) {
+        this.homepageurl = homepageurl;
+        return this;
+    }
+
+
```

Comment:
As SW360Release is a representation of a domain object in SW360, I find it a bit strange that it has a property not supported by SW360. Wouldn't it be better to use the _additional data_ mechanism provided by this class for this purpose? That way the homepage URL for the release would even be persisted in SW360.

Reply:
That is a good idea, thanks.